source/UsersServer/app/users.py

```diff
@@ -0,0 +1,55 @@
+from typing import Optional
+
+from beanie import PydanticObjectId
+from fastapi import Depends, Request
+from fastapi_users import BaseUserManager, FastAPIUsers
+from fastapi_users.authentication import (
+    AuthenticationBackend,
+    BearerTransport,
+    JWTStrategy,
+)
+from fastapi_users.db import BeanieUserDatabase, ObjectIDIDMixin
+
+from app.db import User, get_user_db
+
+SECRET = "SECRET"
```

Comment:
please add a todo comment to check how to properly generate and safely store the secret